src/main/java/org/team1540/bunnybotTank2023/RamseteConfig.java

```diff
@@ -0,0 +1,40 @@
+package org.team1540.bunnybotTank2023;
+
+import com.pathplanner.lib.PathConstraints;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.RamseteController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.kinematics.DifferentialDriveKinematics;
+
+import static org.team1540.bunnybotTank2023.Constants.*;
+
+public class RamseteConfig {
+    // Kinematics
+    public static final DifferentialDriveKinematics driveKinematics = new DifferentialDriveKinematics(
+            DrivetrainConstants.TRACK_WIDTH
+    );
+
+    // PID controllers
+    public static final PIDController leftPID = new PIDController(
```

Comment:
Are you copying this from somewhere or was this your idea? I'd say pid controllers should generally not be static